qiskit_experiments/__init__.py

```diff
@@ -17,9 +17,55 @@
 
 .. currentmodule:: qiskit_experiments
 
-*Note: This package is still under active development, there will be breaking
-API changes, and re-organization of the package layout.*
+.. warning::
```

Comment:
👍 what about adding a link to experiment issues so that user can find where to report problems?